x/evm/config/configurator.go

```diff
@@ -0,0 +1,55 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+//
+// The config package provides a convinient way to modify x/evm params and values.
+// Its primary purpose is to be used during application initialization.
+
+package config
+
+import (
+	"golang.org/x/exp/slices"
+
+	"github.com/evmos/evmos/v18/x/evm/core/vm"
+	"github.com/evmos/evmos/v18/x/evm/types"
+)
+
+// EvmConfigurator allows to extend x/evm module configurations. The configurator modifies
+// the EVM before starting the node, and all the validation are left to the InitGenesis of
+// the module.
+type EVMConfigurator struct {
```

Comment:
This is a feature to allow external users of evmOS to provide a custom default config of the x/evm. It is not associated with a node config but with evm one. Do you think it is not required?

Reply:
I think that it's fine to have the same logic to apply our EIPs and users EIPs, the design is different from the one in Notion because we are not passing the activators to the `evm.Keeper`

Reply:
Yeah it is not exactly the same design we have on Notion because in this way we can have a unified approach to define both new activators and add them to the default one. If you prefer the Notion one I can update it!